File: cpp/bc1794.cpp
#include <iostream>

using namespace std;

int main()
{
	int tRo, lA, lB, sA, sB;
	string res;
	
	cin >> tRo;
	cin >> lA >> lB >> sA >> sB;
	
	res = (tRo >= lA && tRo <= lB) && (tRo >= sA && tRo <= sB)? "possivel" : "impossivel";
	
	cout << res << endl;
}

File: python/bc1794.py
tRo = int(input())

fLi = input().split(" ")
lA = int(fLi[0])
lB = int(fLi[1])

sLi = input().split(" ")
sA = int(sLi[0])
sB = int(sLi[1])

res = "possivel" if (tRo >= lA and tRo <= lB) and (tRo >= sA and tRo <= sB) else "impossivel"

print(res)